app/src/main/res/values/strings.xml

```diff
@@ -607,6 +616,10 @@
     <string name="settings_autostart_bootup_heading">Auto-start on power-up (experimental)</string>
     <string name="settings_autostart_bootup_desc">On device power-up, start RethinkDNS if it was running before shut down.</string>
 
+    <string name="settings_network_header">NETWORK</string>
```

Comment:
are all other section labels upper-case in strings.xml or are _styled_ to upper-case?

Reply:
Other than **VPN & Proxy**. Everything is in upper-case in strings.xml